draft/0002-authentication.md

```diff
@@ -172,7 +186,8 @@ can be marked as authenticated by placing an `Authenticated` list item.
 
 ### Actions
 
-You can mark a specific action as authenticated using the following:
+You can mark a specific action include all example requests as authenticated
+using the following:
```

Comment:
Actually this description part is a different format compared to the descriptions under `Resource` and `Resource groups`.

This has `You can mark` (active form), while they have `can be marked` (passive form)